boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -140,31 +139,29 @@ mb1:
 
 mb2:
     /* See if the boot device was passed in */
-    test dword ptr ds:[ebx + MB_INFO_FLAGS_OFFSET], MB_INFO_FLAG_BOOT_DEVICE
+    test dword ptr [rbx + MB_INFO_FLAGS_OFFSET], MB_INFO_FLAG_BOOT_DEVICE
```

Comment:
Does it need rbx here? At line 127 above, ebx is used and seems to work.

Reply:
see the defines in the start of the file

Reply:
Yes. i will fix the `mbchk_command_line`